### PR TITLE
Readme for model

### PR DESCRIPTION
**Pull request by mosesdaudu001**

### CODE REVIEW

File: ivy_models/bert/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    </a>
+    <a href="https://github.com/unifyai/models/actions?query=workflow%3Anightly-tests">
+        <img class="dark-light" style="float: left; padding-right: 4px; padding-bottom: 4px;" src="https://github.com/unifyai/models/actions/workflows/nightly-tests.yml/badge.svg">
+    </a>
+    <a href="https://discord.gg/G4aR9Q7DTN">
+        <img class="dark-light" style="float: left; padding-right: 4px; padding-bottom: 4px;" src="https://img.shields.io/discord/799879767196958751?color=blue&label=%20&logo=discord&logoColor=white">
+    </a>
+    <br clear="all" />
+
+Bert
```
Comment (juliagsy) [inline ivy_models/bert/README.rst:27]: small details, I'd suggest following the official styles, ie, all caps

File: ivy_models/unet/README.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    </a>
+    <a href="https://github.com/unifyai/models/actions?query=workflow%3Anightly-tests">
+        <img class="dark-light" style="float: left; padding-right: 4px; padding-bottom: 4px;" src="https://github.com/unifyai/models/actions/workflows/nightly-tests.yml/badge.svg">
+    </a>
+    <a href="https://discord.gg/G4aR9Q7DTN">
+        <img class="dark-light" style="float: left; padding-right: 4px; padding-bottom: 4px;" src="https://img.shields.io/discord/799879767196958751?color=blue&label=%20&logo=discord&logoColor=white">
+    </a>
+    <br clear="all" />
+
+Unet
```
Comment (juliagsy) [inline ivy_models/unet/README.rst:27]: small details, I'd suggest following the official styles
ps: I'm not sure if it is all caps official, but if this is already the official style, please feel free to ignore this comment

**Review by juliagsy — COMMENTED** [review]
Small details! Once they are addressed, lgtm and feel free to merge, thanks!

File: ivy_models/bert/README.rst (hunk 1 above)
Comment (mosesdaudu001) [reply ivy_models/bert/README.rst:27]: Done

File: ivy_models/unet/README.rst (hunk 2 above)
Comment (mosesdaudu001) [reply ivy_models/unet/README.rst:27]: The official is actually `U-Net` and I have made necessary changes as requested

**Pull request merged**